resources/addons-namespaces/main.yaml

```
@@ -0,0 +1,21 @@
+ocs-converged-dev: openshift-storage
```

To valid yaml, there needs to be a root element:
```suggestion
---
addon-namespaces:
   ocs-converged-dev: openshift-storage
```

---

I fixed the script to generate a nested dict, so the yaml it generates has a root element.